app/src/main/java/com/codesoom/assignment/handlers/TaskHttpHandler.java

```diff
@@ -0,0 +1,163 @@
+        String requestMethod = exchange.getRequestMethod();
+        String requestURI = exchange.getRequestURI().getPath();
+        String requestBody = getRequestBody(exchange.getRequestBody());
+
+        System.out.println(requestMethod + " " + exchange.getRequestURI());
+
+        Headers responseHeaders = exchange.getResponseHeaders();
+        responseHeaders.add("Content-Type", "application/json");
+
+        ResponseDto responseDto = handleRequest(requestMethod, requestURI, requestBody);
+
+        OutputStream outputStream = exchange.getResponseBody();
+        exchange.sendResponseHeaders(responseDto.getHttpStatus(), responseDto.getResponseBody().getBytes(StandardCharsets.UTF_8).length);
+        outputStream.write(responseDto.getResponseBody().getBytes(StandardCharsets.UTF_8));
+        outputStream.flush();
+        outputStream.close();
+    }
+
+    private ResponseDto handleRequest(String httpMethod, String requestURI, String requestBody) throws IOException {
+        if (PATH.equals(requestURI) && HttpMethod.GET.name().equals(httpMethod)) {
+            return getTasks();
+        }
+        if (PATH.equals(requestURI) && HttpMethod.POST.name().equals(httpMethod)) {
+            return createTask(toTask(requestBody));
+        }
+        if (isMatch(requestURI) && HttpMethod.GET.name().equals(httpMethod)) {
+            return getTask(getPathVariable(requestURI));
+        }
+        if (isMatch(requestURI) && HttpMethod.PATCH.name().equals(httpMethod)) {
+            return updateTask(getPathVariable(requestURI), toTask(requestBody));
+        }
+        if (isMatch(requestURI) && HttpMethod.DELETE.name().equals(httpMethod)) {
+            return deleteTask(getPathVariable(requestURI));
+        }
+
+        return handleError(HTTP_STATUS_NOT_FOUND, "Not Found");
+    }
+
+
+    private ResponseDto getTasks() throws IOException {
+        return new ResponseDto(HTTP_STATUS_OK, tasksToJSON(taskService.getTasks()));
+    }
+
+    private ResponseDto createTask(TaskDto requestDto) throws IOException {
+        final Task newTask = taskService.addTask(requestDto);
+        return new ResponseDto(HTTP_STATUS_CREATE, taskToJSON(newTask));
+    }
+
+    private ResponseDto getTask(Long id) throws IOException {
+        final Task findTask = taskService.findTaskById(id);
+        return new ResponseDto(HTTP_STATUS_OK, taskToJSON(findTask));
+    }
+
+    private ResponseDto updateTask(Long id, TaskDto requestDto) throws IOException {
+        final Task updatedTask = taskService.updateTaskById(id, requestDto);
+        return new ResponseDto(HTTP_STATUS_OK, taskToJSON(updatedTask));
+    }
+
+    private ResponseDto deleteTask(Long id) {
+        taskService.deleteTaskById(id);
+        return new ResponseDto(HTTP_STATUS_OK, "");
+    }
+
+    private ResponseDto handleError(int httpStatus, String message) {
+        return new ResponseDto(httpStatus, message);
+    }
+
+    /**
+     * 모든 Task 목록을 JSON 형식으로 반환한다.
+     * */
```

Comment:
블록 주석에서 닫는 쪽에서 별은 한 번만 써도 됩니다.
```suggestion
    /**
     * 모든 Task 목록을 JSON 형식으로 반환한다.
     */
```